spec/system/admin_rewards_from_csv_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+require 'factory_bot_rails'
+
+RSpec.describe 'Import rewards from a CSV', type: :system do
+  let(:admin_user) { create(:admin_user) }
+
+  before do
+    driven_by(:rack_test)
+    sign_in admin_user
+  end
+
+  context 'when importing rewards from csv file' do
+    it 'imports reward from csv file' do
+      expect(Reward.count).to eq 0
+      visit dashboard_admin_users_pages_path
+      click_link 'Manage Rewards'
+
+      click_link 'Import rewards from CSV'
+      attach_file(File.absolute_path('./spec/files/test1.csv'))
+      click_button 'Import Rewards'
+
+      expect(page).to have_content 'Rewards was successfully imported.'
+      expect(Reward.count).to eq 1
+      expect(Reward.first.title).to eq 'RewardTitle'
+      expect(Reward.first.description).to eq 'RewardDescription'
+      expect(Reward.first.price).to eq 3
+      expect(Reward.first.category.title).to eq 'Category1'
+      expect(page).to have_content 'RewardTitle'
+      expect(page).to have_content 'RewardDescription'
+      expect(page).to have_content '3'
+      expect(page).to have_content 'Category1'
+
+      # WHEN REWARD WITH IMPORTED TITLE EXISTS ITS DESCRIPTION, PRICE AND CATEGORY SHOULD BE UPDATED
```

Comment:
It could be a new test, like `it 'updates existing rewards during import'`. Then you don't need this comment :)

Reply:
Done! 👍